### ISSUE TITLE
WordPress Multisite - htaccess redirects overwritten

### ISSUE THREAD

**Issue by edellingham** [johngodley/redirection#3352]
### Summary

We're using the plugin with WordPress Multisite.  We're experiencing an issue with LiteSpeed Cache caching the page after the redirect is set after a few hours, causing the redirect to fail.  I've tested with the 'no-cache' header in 'Options', but LiteSpeed Cache is still caching the page hours later and causing the redirect to fail.

I'm trying to use the htaccess feature, which I think will solve the issue, but when I create a redirect in Site A and then create a redirect in Site B, Site B will replace the Site A redirect once saved.  I'd expect it to add each site's redirect to the htaccess file instead of replacing the previously existing one.

We're using the latest plugin on DB 4.2

Is there a workaround?

I appreciate your help and this plugin!

**Comment by johngodley** [comment]
No, the plugin doesn't support multiple versions writing to the same file. You could install the plugin on your network and manage everything from one place, or you could manually export the redirects and add them yourself.
